src/components/ThreeBoxUpdate.re

```diff
@@ -571,9 +571,14 @@ module Main = {
              "to validate your identity on twitter."->React.string
            </Rimble.Text>
            <br />
-           <Rimble.Button onClick={_e => {setIsIntegarted3Box(_ => true)}}>
-             {React.string("Try Experimental 3box integration")}
-           </Rimble.Button>
+           <Rimble.Text>
```

Comment:
Since this is more experimental made this into text (rather than a full button).